packages/css/src/table.css

```diff
@@ -12,7 +12,7 @@
   --dsc-table-divider-border-width: calc(var(--ds-border-width-default) + 1px);
   --dsc-table-header-background--hover: var(--ds-color-surface-tinted);
   --dsc-table-header-background--sorted: var(--ds-color-background-tinted);
-  --dsc-table-header-background: var(--ds-color-background-default);
+  --dsc-table-header-background: transparent;
```

Comment:
as mentioned on slack, this makes sticky header also transparent.

<img width="390" alt="image" src="https://github.com/user-attachments/assets/0462e08a-d867-4f96-9cf5-4791887fc65a" />

Reply:
@Febakke @Thunear

Reply:
Suggest keep it transparent for now and then add a background later if someone has a problem with it. Add a section in the docs about needing to add a background color on `thead` that matches your background color if you plan on using `stickyHeader`?

Reply:
I disagree, it makes no sense for us to release an unfinished feature.
I'd rather add a bakground only to the header, or remove the sticky header feature completely

Reply:
Jepp, lets add a `background-default` color as default, since you either way have to change a background color if it does not match, this way it may be ok for most people, and they don't need to change the color.